src/main/java/seedu/address/logic/parser/module/ModuleAddCommandParser.java

```diff
@@ -13,7 +13,7 @@
 import seedu.address.logic.parser.ParserUtil;
 import seedu.address.logic.parser.Prefix;
 import seedu.address.logic.parser.exceptions.ParseException;
-import seedu.address.model.module.Description;
```

Comment:
Refactored?
Or is this a common Description object?

Reply:
Yeah I refactored, since Task also has a Description (wrapper for String class).